### PR TITLE
Added Decentralized Finance (DeFi) Projects in Web3 Resource List

### LINKED ISSUE #205: 💡 [FEATURE] - Addition of "Decentralized Finance (DeFi) Projects" in Web3 Resource List
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

I would like to request the addition of a new section titled "Decentralized Finance (DeFi) Projects" in the existing Web3 Resource List. This section aims to provide users with information about notable decentralized finance platforms and protocols within the Web3 ecosystem.

### Explain your solution

The proposed "Decentralized Finance (DeFi) Projects" section will enhance the README by offering users insights into the diverse range of financial applications available in the Web3 space. These projects play a crucial role in reshaping traditional finance by leveraging blockchain technology and smart contracts to offer decentralized financial services such as lending, borrowing, trading, and yield farming. Including this section will benefit users who are interested in exploring the diverse range of financial applications available in the Web3 space. It will provide them with valuable insights into popular DeFi projects, enabling them to learn more about decentralized finance and participate in the growing DeFi ecosystem.

### Any alternative approaches/features

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by SkSadaf**
**PR Title :**
- Added "Decentralized Finance (DeFi) Projects" in Web3 Resource List

**Fixes Issue**
-Closes #205 

**Description:** 
This pull request adds a new section titled "Decentralized Finance (DeFi) Projects" to the existing Web3 Resource List. The section aims to provide learners with valuable insights into notable decentralized finance platforms and protocols within the Web3 ecosystem.
Key Features Added:
Project Name
Description
Category/Type
Launch Date
TVL (Total Value Locked)
Founder/Team
Audited

**Checklist:**
Before submitting your pull request, ensure that you have completed the following tasks: 
- [YES ] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [YES ] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

File: Web3/README.md [hunk 1]
```diff
@@ -233,5 +235,88 @@ Analytics.</td>
   </tr>
 </table>
 
-
+### Decentralized Finance (DeFi) Projects
```
Comment (jfmartinz) [inline Web3/README.md:238]:
```suggestion
### Decentralized Finance (DeFi) 
```

File: Web3/README.md [hunk 2]
```diff
@@ -7,7 +7,9 @@
 - [Blockchain Technologies](#blockchain-technologies)<br>
 - [News](#news)<br>
 - [Community](#community)<br>
-- [Youtube Channels](#youtube-channels)
+- [Youtube Channels](#youtube-channels)<br>
+- [Decentralized Finance (DeFi) Projects](#Decentralized-Finance-(DeFi)-Projects)
```
Comment (jfmartinz) [inline Web3/README.md:11]:
```suggestion
- [Decentralized Finance (DeFi)](#Decentralized-Finance-(DeFi))
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]

File: Web3/README.md (hunk 2 above)
Comment (SkSadaf) [reply Web3/README.md]: I have made the proposed change, please review and let me know if any changes are to be made.

**Comment by SkSadaf** [comment]
I have made the proposed change, please review and let me know if any changes are to be made.

**Review by jfmartinz — APPROVED** [review]

**Pull request merged**